### PR TITLE
Update decommission get-many for change plan

### PR DESCRIPTION
**Pull request by juliasaveliff**
image

### CODE REVIEW

File: rackcity/utils/change_planner_utils.py [hunk 1]
```diff
@@ -143,7 +164,67 @@ def get_page_of_serialized_assets(all_assets, query_params):
         return all_assets[start:end]
 
 
-def get_many_assets_response_for_cp(request, change_plan):
+def get_many_assets_response_for_cp(request, change_plan, decommissioned=False):
```
Comment (juliasaveliff) [inline rackcity/utils/change_planner_utils.py]: git is up to something. this entire method already existed. just added the `decommissioned=False` parameter. will comment on other lines that changed

File: rackcity/utils/change_planner_utils.py [hunk 2]
```diff
@@ -143,7 +164,67 @@ def get_page_of_serialized_assets(all_assets, query_params):
...
+                request.data,
+            )
+        if filter_failure_response:
+            return filter_failure_response
+    else:
+        assets, assetsCP, filter_failure_response = \
+            get_filtered_decommissioned_assets_for_cp(
+                change_plan,
+                request.data,
+            )
```
Comment (juliasaveliff) [inline rackcity/utils/change_planner_utils.py]: this is new

File: rackcity/views/rackcity_utils.py [hunk 3]
```diff
@@ -176,16 +178,19 @@ def validate_location_modification(data, existing_asset, user, change_plan=None)
...
-            )
+        )
     except ObjectDoesNotExist:
         return (None, JsonResponse(
             {
                 "failure_message":
-                    Status.CREATE_ERROR.value + GenericFailure.INTERNAL.value,
+                    Status.ERROR.value +
+                    "Change plan" +
+                    GenericFailure.DOES_NOT_EXIST.value,
```
Comment (juliasaveliff) [inline rackcity/views/rackcity_utils.py]: Improved this error message

File: rackcity/utils/change_planner_utils.py [hunk 4]
```diff
@@ -96,6 +96,27 @@ def detect_conflicts_cp(sender, **kwargs):
     ).update(asset_conflict_asset_number=asset)
 
 
+def get_filtered_decommissioned_assets_for_cp(change_plan, data):
+    decommissioned_assets = DecommissionedAsset.objects.all()
+    decomissioned_assets_cp = AssetCP.objects.filter(
```
Comment (benauriemma) [inline rackcity/utils/change_planner_utils.py]: Do we not have a DecommissionedAssetCP model?? Wild

Comment (juliasaveliff) [reply rackcity/utils/change_planner_utils.py]: ah ha ha ...

Comment (juliasaveliff) [reply rackcity/utils/change_planner_utils.py]: we elected to not have one.

**Pull request merged**